xarray/core/dataset.py

```diff
@@ -4537,7 +4537,7 @@ def _set_sparse_data_from_dataframe(
         idx = dataframe.index
         if isinstance(idx, pd.MultiIndex):
             coords = np.stack([np.asarray(code) for code in idx.codes], axis=0)
-            is_sorted = idx.is_lexsorted
+            is_sorted = idx.is_lexsorted()
```

Comment:
Good catch! This is a little embarrassing :)

Reply:
:) I blame Pandas API. `Index.is_sorted` and `Index.is_monotonic` are both properties! So expecting `is_lexsorted` to be a property is totally reasonable!